cdflow_commands/config.py

```diff
@@ -49,7 +57,9 @@ def load_service_metadata():
                 metadata['TYPE'],
                 metadata['REGION'],
                 metadata['ACCOUNT_PREFIX'],
-                metadata.get('ECS_CLUSTER', 'default')
+                metadata.get('ECS_CLUSTER', 'default'),
+                metadata.get('HANDLER', ''),
+                metadata.get('RUNTIME', '')
```

Comment:
+1 for me. 

My only concern is that we are coupling the format of these configuration files to the way they are consumed (terraform variables as maps). That might restrict us and drive us to make decisions based on the convenience of the implementation details. 

But I as this is already case for platform-config and so on, so I guess repeat the pattern is not bad.